platforms/win32/vm/sqWin32Prefs.c

```diff
@@ -207,8 +207,17 @@ void LoadPreferences()
   }
 
   /* get window title from ini file */
+#ifdef UNICODE
+  {
+    TCHAR windowTitleT[MAX_PATH];
+    GetPrivateProfileString(U_GLOBAL, TEXT("WindowTitle"),
+      TEXT(""), windowTitleT, MAX_PATH, squeakIniName);
+    WideCharToMultiByte(CP_UTF8, 0, windowTitleT, -1, windowTitle, MAX_PATH, NULL, NULL);
```

Comment:
This needs a check

Reply:
Definitely! this can only work with -DUNICODE... I will emit a new commit.

Reply:
Wait, I have been abused by the too-short context: this piece of code is precisely protected with a `#ifdef UNICODE`, so finally I think it is correct.

We may switch to using the W version unconditionnally, but for that, `squeakIniName` must also be UTF-16 encoded. This is part of larger changes that I have cooking and did not commit yet.

Reply:
OK!